app/models/report.rb

```diff
@@ -27,6 +27,7 @@ def self.inherited(child)
       scoped_search :relation => :hostgroup,   :on => :title, :complete_value => true, :rename => :hostgroup_title
 
       scoped_search :on => :reported_at, :complete_value => true, :default_order => :desc, :rename => :reported, :only_explicit => true
+      scoped_search :on => :host_id,     :complete_value => true, :only_explicit => true
```

Comment:
@ares made a good comment on the similar GH-4993 - please make complete_value false so it doesn't auto-suggest all host ids.